x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -143,6 +142,30 @@ export function XYConfigPanel(props: VisualizationProps<State>) {
         </EuiFormRow>
       )}
 
+      <EuiFormRow
+        label={i18n.translate('xpack.lens.xyChart.splitSeries', {
+          defaultMessage: 'Split series',
+        })}
+      >
+        <MultiColumnEditor
+          accessors={state.splitSeriesAccessors}
+          datasource={datasource}
+          dragDropContext={props.dragDropContext}
+          onAdd={accessor =>
+            setState({ ...state, splitSeriesAccessors: [...state.splitSeriesAccessors, accessor] })
+          }
+          onRemove={accessor =>
+            setState({
+              ...state,
+              splitSeriesAccessors: state.splitSeriesAccessors.filter(col => col !== accessor),
+            })
+          }
+          filterOperations={op => op.isBucketed}
```

Comment:
I think it's a better experience to remove dates as filter options, so `op => op.isBucketed && op.dataType !== 'date'`

Reply:
Agreed, though eventually, it makes sense to split on dates (e.g. split by month or weekday or something of that sort).